File: Iniciante/URI 1017.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val tempo = leitor.nextInt()
        val velocidade = leitor.nextInt()
        val kilometros = tempo * velocidade
        val media = kilometros / 12.0
        printf("%.3f\n", media)
    }
	
}

File: Iniciante/URI 1017.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val tempo: Double = leitor.nextDouble();
    val velocidade: Double = leitor.nextDouble();
    val kilometros: Double = tempo * velocidade;
    val media: Double = kilometros / 12;
    
    println(String.format("%.3f", media)); 
}